src/containers/shared/components/Transaction/MPTokenIssuanceCreate/parser.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  MPTokenIssuanceCreate,
+  MPTokenIssuanceCreateInstructions,
+} from './types'
+import { TransactionParser } from '../types'
+import { convertHexToString } from '../../../../../rippled/lib/utils'
+import { convertHexToBigInt } from '../../../utils'
+
+export const parser: TransactionParser<
+  MPTokenIssuanceCreate,
+  MPTokenIssuanceCreateInstructions
+> = (tx, meta) => ({
+  issuanceID: meta.mpt_issuance_id,
+  metadata: tx.MPTokenMetadata
```

Comment:
Should we also deserialize the JSON string and possibly show in 2 columns (key - value) for readability?

Reply:
We can also just show the metadata on the MPT page only

Reply:
I think that's a good idea! But currently the metadata format hasn't been standardized yet, and the one shown in the screenshot is just an example. We can def make this improvement in the future.